src/utils.c

```diff
@@ -59,13 +63,17 @@ GthreeObject *
 load_model (const char *name)
 {
   GError *error = NULL;
+  char *path;
   g_autoptr(GFile) base = NULL;
   g_autoptr(GFile) file = NULL;
   g_autoptr(GFile) parent = NULL;
   g_autoptr(GBytes) bytes = NULL;
   g_autoptr(GthreeLoader) loader = NULL;
 
-  base = g_file_new_for_path ("../models");
+  path = g_getenv("GNOME_HEXGL_DATADIR");
```

Comment:
Missing space between `g_getenv` and opening parenthesis.

Reply:
yup